File: Csharp/136.cs
public class Solution {
    public int SingleNumber(int[] nums) {
        
        List<int> listInt = new List<int>();
        for(int i = 0; i < nums.Length; i++){
            if(listInt.Contains(nums[i])){
                listInt.Remove(nums[i]);
            } else {
                listInt.Add(nums[i]);
            }
        }
        
        return listInt[0];
    }
}

File: JS/136.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var singleNumber = function(nums) {
    for(let i =0; i < nums.length; i++){
        let arr = nums;
        arr[i] = null;
        console.log(arr);
    }
    return 0
};